src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCDetailsPage.java

```diff
@@ -15,7 +16,7 @@
  * SFCDetailsPage.java
  * Class that represents the page where details of an contact are displayed.
  */
-public class SFCDetailsPage extends SalesForceConnection {
+public class SFCDetailsPage extends HomeBasePage {
     public static final WebDriverWait WAITER = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)